core/modules/node/node.module

```diff
@@ -1322,7 +1322,7 @@ function node_views_api() {
  * @param SearchQuery|SelectQueryExtender $query
  *   A query object that has been extended with the Search DB Extender.
  */
-function _node_rankings(SelectQueryExtender $query) {
+function _node_rankings(SelectQueryExtender $query, $keywords = NULL) {
   if ($ranking = module_invoke_all('ranking')) {
```

Comment:
Would need to pass `$keywords` here, or use `module_implements()`

Reply:
yes, see line 1434

Reply:
Ah, right. Missed this one :)